### PR TITLE
Add special tags information tooltip to content asset tags field

### PR DESCRIPTION
**Pull request by mttdffy**
WORKAREA-99

### CODE REVIEW

File: admin/config/locales/en.yml [hunk 1]
```diff
@@ -746,6 +746,21 @@ en:
...
+              <strong>Favicons:</strong> If you want to use an image asset as a favicon for the site,
+              you can add the <code>favicon</code> tag. Additionally, you can specify use for a
+              specific size favicon by tagging it with <code>favicon-HxW</code>. For example, a tag of
+              <code>favicon-32x32</code> will indicate the asset should be used for the favicon when
+              it's size is 32x32.
+            </p>
+            <p>
+              <strong>Open Graph:</strong> You can specify what image asset to use by default for the
+              open graph image of your content pages by tagging it with the <code>og-default</code> tag.
+              When shared on Facebook, that asset will display if there is no open graph image specified
```
Comment (bencrouse) [inline admin/config/locales/en.yml]:
```suggestion
              When shared on social media, that asset will display if there is no open graph image specified
```

File: admin/config/locales/en.yml [hunk 2]
```diff
@@ -746,6 +746,21 @@ en:
...
           tags_label: Tags
           tags_note: 'Comma separated: just, like, this'
+          tags_tooltip_html: |
+            <p>You can specify tags on an asset to mark them for special use cases.</p>
+            <p>
+              <strong>Favicons:</strong> If you want to use an image asset as a favicon for the site,
+              you can add the <code>favicon</code> tag. Additionally, you can specify use for a
+              specific size favicon by tagging it with <code>favicon-HxW</code>. For example, a tag of
+              <code>favicon-32x32</code> will indicate the asset should be used for the favicon when
+              it's size is 32x32.
```
Comment (bencrouse) [inline admin/config/locales/en.yml]: Similar to explaining sharing below, should we briefly explain why you might use the different sizes?

File: admin/config/locales/en.yml [hunk 3]
```diff
@@ -746,6 +746,21 @@ en:
           save_asset: Save Asset
           tags_label: Tags
           tags_note: 'Comma separated: just, like, this'
+          tags_tooltip_html: |
+            <p>You can specify tags on an asset to mark them for special use cases.</p>
+            <p>
+              <strong>Favicons:</strong> If you want to use an image asset as a favicon for the site,
```
Comment (bencrouse) [inline admin/config/locales/en.yml]:
```suggestion
              <strong>Favicons:</strong> If you want to use an image asset as the default favicon for the site,
```

File: admin/config/locales/en.yml (hunk 1 above)
Comment (mttdffy) [reply admin/config/locales/en.yml]: is this true? I thought Facebook was the only use for open graph tags.

File: admin/config/locales/en.yml [hunk 4]
```diff
@@ -746,6 +746,21 @@ en:
           save_asset: Save Asset
           tags_label: Tags
           tags_note: 'Comma separated: just, like, this'
+          tags_tooltip_html: |
+            <p>You can specify tags on an asset to mark them for special use cases.</p>
```
Comment (chriscressman) [inline admin/config/locales/en.yml]: I'm not sure what markup is best of this context, but regarding content, how about something like this?:

Use the following tags to set an asset as a favicon or Open Graph image for your site:

"favicon" - The default favicon
"favicon-<var>W</var>x<var>H</var>", e.g. "favicon-32x32" - The favicon for the given size
"og-default" - The default Open Graph image

The default favicon is used when a device does not request a specific size or when the requested size is not available.
The default Open Graph image is used when sharing a content page that does not specify an Open Graph image.

File: admin/config/locales/en.yml (hunk 1 above)
Comment (bencrouse) [reply admin/config/locales/en.yml]: I know Slack also uses it, maybe others.

File: admin/config/locales/en.yml (hunk 4 above)
Comment (bencrouse) [reply admin/config/locales/en.yml]: sounds good to me.

**Pull request merged**